docarray/array/storage/annlite/backend.py

```diff
@@ -80,7 +81,7 @@ def _init_storage(
         subindex_configs: Optional[Dict] = None,
         **kwargs,
     ):
-
+        config = copy.deepcopy(config)
```

Comment:
why deepcopy here? the `data_path` doesn't change

Reply:
we need it because in some cases you pass the same config object to multiple das. So even though data_path is randomly generated for some DA, it will be re-used for the next one, implicitly.
That's why I try to enforce the same pattern for all backends to avoid this kind of issues